.github/workflows/sauce-ie11.yaml

```diff
@@ -23,6 +23,12 @@ jobs:
           node-version: 16
           cache: 'npm'
           cache-dependency-path: '**/package-lock.json'
+      
+      - uses: saucelabs/sauce-connect-action@v2
```

Comment:
This shouldn't be here, right?

Reply:
yes but that's from a totally different commit

Reply:
good catch but i have no idea why that's there

Reply:
done